src/app/pipes/sort.pipe.ts

```diff
@@ -24,7 +24,20 @@ export class SortPipe implements PipeTransform {
       second = 1;
     }
 
-    if (parameters === 'datasetAccession') {
+    // alternative sorting method, need to be discussed/decided first
+    // return value.sort(function (a, b) {
+    //   if (a['standard'] === b['standard']) {
+    //     if (parameters === 'datasetAccession' || parameters === 'accession') {
+    //       const one = parseInt(a[parameters].match(/[a-zA-z]+([0-9]+)$/)[1], 10);
+    //       const two = parseInt(b[parameters].match(/[a-zA-z]+([0-9]+)$/)[1], 10);
+    //       return one > two ? first : (two > one ? second : 0);
+    //     } else {
+    //       return a[parameters] > b[parameters] ? first : (b[parameters] > a[parameters] ? second : 0);
+    //     }
+    //   }
+    //   return a['standard'] > b['standard'] ? 1 : -1;
+    // });
```

Comment:
No need to have all this code commented out, you can add # TODO comment for actions that we should consider in future

Reply:
done in 82bf714